django/santropolFeast/member/templates/client/base.html

```diff
@@ -5,26 +5,75 @@
 
 {% block title %}Client{% endblock %}
 
+{% block extrahead %}
+<link href="{% static 'css/calendar.min.css' %}" type="text/css" rel="stylesheet">
+{% endblock %}
+
```

Comment:
You don't need to import the calendar css anymore, as it is imported by the main base.html template.